packages/clippy-components/src/clippy-font-combobox/index.ts

```diff
@@ -0,0 +1,78 @@
+import { html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { ref } from 'lit/directives/ref.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { ClippyCombobox } from '../clippy-combobox';
+
+type Option = {
+  label: string;
+  value: string | Array<string>;
+  cssUrl?: string;
+};
+
+const tag = 'clippy-font-combobox';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    [tag]: ClippyFontCombobox;
+  }
+}
+
+@customElement(tag)
+export class ClippyFontCombobox extends ClippyCombobox<Option> {
+  #additional?: Option[];
+  #intersectionObserver?: IntersectionObserver;
+
+  override readonly filter = (option: Option) => {
+    const label = `${option.label}`; // Use as string
+    return label.toLowerCase().includes(this.query.toLowerCase());
+  };
+
+  override async fetchAdditionalOptions(query: string): Promise<Option[]> {
+    this.#additional = this.#additional ?? (await import('./external').then(({ default: items }) => items));
+    const options = this.#additional.filter((option) => option.label.includes(query));
+
+    if (options.length) {
+      this.#intersectionObserver =
+        this.#intersectionObserver ??
+        new IntersectionObserver(this.#loadFontsInView, {
+          root: this.querySelector('[role=listbox]'),
```

Comment:
nitpick: feels like this could be initialized only once in a query decaoratted var https://lit.dev/docs/api/decorators/#query